ci-operator/step-registry/gather/audit-logs/gather-audit-logs-commands.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+dest_dir=$(mktemp --dir --tmpdir="$ARTIFACT_DIR")
+oc adm must-gather --dest-dir="$dest_dir" --image=quay.io/openshift/origin-must-gather -- /usr/bin/gather_audit_logs
```

Comment:
Why are we hard-coding `--image`?  Can't we use the must-gather referenced from the cluster's release image?